01-Login/src/app/app.module.ts

```diff
@@ -44,4 +46,10 @@ export function hljsLanguages() {
   providers: [],
   bootstrap: [AppComponent]
 })
-export class AppModule {}
+export class AppModule {
+  constructor() {
+    library.add(faUser);
```

Comment:
The "implicit reference" mode requires that the icons are all declared up-front, rather than per-component.